examples/onlineboutiquemicroservice/proto-file/demo.proto

```diff
@@ -0,0 +1,261 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
Let's add reference like https://github.com/ballerina-platform/module-ballerina-grpc/blob/1b70299d32412bd2b943ce9214f846f57f481e29/examples/routeguide/proto-file/route_guide.proto#L15

Reply:
Added in https://github.com/ballerina-platform/module-ballerina-grpc/pull/392/commits/56b949cc1bb433c00a38ac181b0b233f12087e9b

Reply:
@dilanSachi shall we add the latest release tag instead of master, because master could change eventually?
Something like:
https://github.com/GoogleCloudPlatform/microservices-demo/blob/v0.2.4/pb/demo.proto

Reply:
Changed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/392/commits/422cfb691f650147d3bbf40cbfbf641695a28fd5